src/common/modal/index.jsx

```diff
@@ -1,8 +1,9 @@
 import { useEffect } from "react";
 import ReactDOM from "react-dom";
 import { GoCheck, GoX } from "react-icons/go";
+import classes from './index.module.css';
 
-const Modal =({ title, show, onClose, onSubmit, children, cname })=> {
+const Modal =({ title, show, onClose, appliedFilterQuery, onClearAppliedFilters, onSubmit, children, cname })=> {
```

Comment:
This modal is being used other places too.. make sure those consuming features are not broken.

Reply:
Well, when Clear All button is clicked, which means filters are removed, so using resetFilter() added by @Angryman18, i am re-rendering the original list of react-plays...

Do you want this or do you just want to remove the filters (like selected options and no. of filters applied shown on filter icon) from modal only and not re-render the react plays

Reply:
we have children in the modal to render any component we want. is it possible to put the button as children along with the existing one instead passing extra props to the modal?

Reply:
How about I use the already existing prop "onClose" and pass a function through that prop that takes
in a string variable (say: actionState) and checks:
if actionState === "Close" : close the modal
else if actionState === "Filter" : resetFilters

Reply:
Okay!
Will look into that.

Reply:
@mayukh551 I still see the parent is refreshing on the clear all link call. Can we please fix that?

Reply:
No that number is important, we can not remove it.